src/routes/chat/ChatHeader/ChatDefaultHeader.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import './ChatHeader.scss';
+
+const ChatDefaultHeader = ({ setHeader }) => {
+  return (
+    <div className="chat__offer__actions columns is-gapless is-mobile">
+      <button
+        type="button"
+        className="column button is-success is-outlined"
+        onClick={() => setHeader(1)}
+      >
+        Make Offer
+      </button>
+      <button type="button" className="column button is-danger is-outlined">
```

Comment:
Same here, now this cancel button doesn't actually do anything I assume but we should also have a `handleCancelOfferClick` function. See `ChatHeader.jsx` comments below as to how we can do it.